simple/stats/config.md

```diff
@@ -110,4 +119,36 @@ The URL of the source.
 
 #### `provenances`
 
-The provenances under a given source should be defined using the `provenances` property as `{provenance-name}:{provenance-url}` pairs.
\ No newline at end of file
+The provenances under a given source should be defined using the `provenances` property as `{provenance-name}:{provenance-url}` pairs.
+
+## `database`
+
+The top-level `database` field can be used to provide information for the database that the 
+importer should write the imported data to.
+
+If a database config is not specified, it writes to a sqlite db file named `datacommons.db` 
+in the output folder.
+
+### Database parameters
+
+#### `type`
+
+The type of database the importer connects to. Currently the importer supports 2 types of databases:
+* `cloudMySql` - A Google Cloud MySQL instance.
```

Comment:
just "cloudsql"? It matches the branding name and I would make it all lowercase for simplicity

Reply:
Done.